gapis/memory/alignof_sizeof.go

```diff
@@ -29,7 +29,7 @@ func AlignOf(t reflect.Type, m *device.MemoryLayout) uint64 {
 		return uint64(m.GetPointer().GetAlignment())
 	case t == tyInt, t == tyUint:
```

Comment:
Aren't you going to hit exactly the same problem with `Int`, `Uint` and `Char`?
Please can you fix all the types together?